docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -84,6 +84,9 @@ using metrics.
       Process statistics, memory consumption, cpu usage, open
       file descriptors.
 
+  `repository`::
+      Statistics about read/write throttling in underlying repositories.
```

Comment:
Let's not restrict ourselves just to stats about throttling at this point in the docs, lest we want to emit more stats later. We can just say it's "statistics about snapshot repositories" here, and describe the exact stats in the response body docs.

Reply:
Thanks for guiding this